packages/protocol/deployments/mainnet-contract-logs-L1.md

```diff
@@ -226,7 +227,9 @@
 
 - ens: `based.taiko.eth`
 - proxy: `0x06a9Ab27c7e2255df1815E6CC0168d7755Feb19a`
-- impl: `0x5110634593Ccb8072d161A7d260A409A7E74D7Ca`
+- impl: `0x4e030b19135869F6fd926614754B7F9c184E2B83`
+- oldFork: `0x904Da4C5bD76f932fE09fF32Ae5D7E3d2A5D2264` at commit `c80373d`
```

Comment:
the oldFork and newFork for 0x4e030b19135869F6fd926614754B7F9c184E2B83 are not the address in this PR!